cms/urls.py

```diff
@@ -113,6 +113,7 @@
     url(r'^group_configurations/{}/(?P<group_configuration_id>\d+)(/)?(?P<group_id>\d+)?$'.format(
         settings.COURSE_KEY_PATTERN), 'group_configurations_detail_handler'),
     url(r'^api/val/v0/', include('edxval.urls')),
+    url(r'^api/tasks/v1/', include('user_tasks.urls')),
```

Comment:
Your call, I just wanted to make sure you were ready to commit to the current schema. =)